### PR TITLE
Feature/eui 89 search results

### PR DESCRIPTION
**Pull request by praveenrajendran101**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/cases/store/actions/case-search.action.spec.ts [hunk 1]
```diff
@@ -1,34 +1,41 @@
-import * as fromSearchCases from './case-search.action';
+
+import { TestBed } from '@angular/core/testing';
+import { BrowserDynamicTestingModule, platformBrowserDynamicTesting } from '@angular/platform-browser-dynamic/testing';
+import { JURISDICTION_SELECTED, JurisdictionSelected, RESET, Reset, APPLIED, Applied } from './case-search.action';
 
 describe('Case Search Actions', () => {
+  // beforeEach(() => {
```
Comment (toireasam) [inline src/cases/store/actions/case-search.action.spec.ts]: can we remove commented out unused code?

File: api/lib/interceptors.ts [hunk 2]
```diff
@@ -33,6 +33,7 @@ export function successInterceptor(response) {
 }
 
 export function errorInterceptor(error) {
+    console.log(error)
```
Comment (adnanakgun) [inline api/lib/interceptors.ts]: Can we remove this console.log?

File: src/cases/store/actions/case-search.action.spec.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/cases/store/actions/case-search.action.spec.ts]: Done, removed it. Thanks

**Comment by fazanki** [comment]
@smogese and @praveenrajendran101 can we discuss do we need to be changing the karma configuration and why. what  are the advantages over the default angular cli stuff

**Comment by udaydenduluri33** [comment]
This has been agreed to be rejected by Mo & Franco

**Pull request closed without merging**